components/AnimatedImage/library/src/commonMain/kotlin/org/jetbrains/compose/animatedimage/AnimatedImage.kt

```diff
@@ -9,6 +9,8 @@ expect class AnimatedImage
 
 /**
  * Loads an [AnimatedImage] from a given path.
+ * The path can be loaded from different sources depending on the platform (only desktop currently):
```

Comment:
> The path

We load an image, not a path :). So we should write "The image can be..."

Reply:
Fixed :P